src/fsharp/service/service.fs

```diff
@@ -2498,6 +2498,13 @@ type BackgroundCompiler(legacyReferenceResolver, projectCacheSize, keepAssemblyC
         | Parser.TypeCheckAborted.Yes  ->  FSharpCheckFileAnswer.Aborted                
         | Parser.TypeCheckAborted.No scope -> FSharpCheckFileAnswer.Succeeded(MakeCheckFileResults(filename, options, builder, scope, dependencyFiles, creationErrors, parseErrors, tcErrors))
 
+    let ClearProjectFilesCache ltok (options: FSharpProjectOptions) =
```

Comment:
nit: lower case

Reply:
I'm leaving it Pascal case because functions right above it are.